core/src/calleeFunctions/index.ts

```diff
@@ -151,6 +152,15 @@ export const getMarketDataRecords = async function (
     return marketDataRecords;
 };
 
+export const getMarketDataRecords = memoizee(_getMarketDataRecords, {
+    promise: true,
+    maxAge: MARKET_DATA_RECORDS_CACHE_MS,
+    length: 3,
```

Comment:
noted, but does not make sense to adjust it now since i would like first to resolve the review-comment: https://github.com/sidestream-tech/unified-auctions-ui/pull/588#pullrequestreview-1298029401